src/solvers/sat/satcheck_minisat2.cpp

```diff
@@ -193,6 +195,10 @@ propt::resultt satcheck_minisat2_baset<T>::do_prop_solve()
   log.statistics() << (no_variables() - 1) << " variables, "
                    << solver->nClauses() << " clauses" << messaget::eom;
 
+  status = statust::ERROR;
```

Comment:
What is going on here? Looks like a separate code clean up task. Let's have a PR for those.